root-packages/runc/build.sh

```diff
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.opencontainers.org/
 TERMUX_PKG_DESCRIPTION="A tool for spawning and running containers according to the OCI specification"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.2.5"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="1.3.0-rc.1"
```

Comment:
It is better to use `1.3.0~rc.1` rather than `1.3.0-rc.1`. Using `1.3.0-rc.1` will break auto-update as `1.3.0-rc.1` > `1.3.0`.

FYI:
https://github.com/termux/termux-packages/blob/1ae8e758471989b12edef8f22a0d0ac17b8663e1/x11-packages/hangover-wine/build.sh#L6-L10

Reply:
good point. thanks